ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -297,6 +298,9 @@ data instance BlockQuery (ShelleyBlock proto era) :: Type -> Type where
   -- longer used (because mainnet has hard-forked to a newer version), it can be
   -- removed.
 
+  GetAccountState 
+    :: BlockQuery (ShelleyBlock proto era) AccountState
```

Comment:
Don't we need `ConwayEraGov`?

Reply:
I checked the source in `cardano-ledger`. It seems we don't need it.

Reply:
Ah, but if we don't need it, then the `requireCG` below should be removed, right?